demo/app/README.md

```diff
@@ -1,10 +1,20 @@
-# Demo/Reference App
+# wallet_sdk_flutter
 
-TODO: Add introduction
+A new Flutter project.
```

Comment:
Looks like we need to add details/intro here. Can we keep the TODO ?

Reply:
fixed